### PR TITLE
Rework GREASE section

### LINKED ISSUE #36: GREASE should be expanded before first use
Something like 
```html
<abbr title="Generate Random Extensions And Sustain Extensibility>GREASE</abbr>
```

### PR DESCRIPTION
**Pull request by yoavweiss**
Closes #36 as well as improves the GREASE section in general

### CODE REVIEW

**Comment by amtunlimited** [comment]
LGTM

File: index.bs [hunk 1]
```diff
@@ -394,19 +394,22 @@ order to reduce the fingerprint the header provides.
...
+Reseting expectations may help to prevent abuse of the UA string's brand in the
+short term, but probably won't help in the long run.  The world of network
+protocol introduced the notion of <abbr title="Generate Random Extensions And
+Sustain Extensibility">GREASE</abbr>. We could borrow from that concept to
+tackle this problem.
+
+User agents SHOULD model `UA` as a set, rather than a single entry. They then
+could encourage standardized processing of the `UA` string, by randomly
+including additional, intentionally incorrect, comma-separated entries with
+arbitrary ordering (similar conceptually to [[I-D.ietf-tls-grease]]). That
```
Comment (amtunlimited) [inline index.bs]: Nit: The reference should probably be on the first instance, in line 404

File: index.bs [hunk 2]
```diff
@@ -394,19 +394,22 @@ order to reduce the fingerprint the header provides.
...
-
-User agents may choose to model `UA` as a set, rather than a single entry. This could encourage
-standardized processing of the `UA` string by 
-Randomly including additional, intentionally incorrect, comma-separated entries with arbitrary
-ordering (similar conceptually to [[I-D.ietf-tls-grease]]) could encourage standardized processing
-if the `UA` string by servers, and reduce the chance that we ossify on a few required strings.
-For example, Chrome 73's `Sec-CH-UA` header might be `"Chrome 73", "NotBrowser 12"`, or
+History has shown us that there are real incentives for user agents to lie
+about their branding in order to thread the needle of sites' sniffing scripts,
+and prevent their users from being blocked by UA-based allow/block lists.
```
Comment (amtunlimited) [inline index.bs]: Possibly include IRL examples (i.e. "KHTML, like Gecko")

File: index.bs (hunk 1 above)
Comment (yoavweiss) [reply index.bs]: cool

File: index.bs (hunk 2 above)
Comment (yoavweiss) [reply index.bs]: I prefer not to go there

**Pull request merged**